src/stactools/cop_dem/constants.py

```diff
@@ -55,17 +55,117 @@
         "application/pdf",
         "Copernicus DEM User handbook",
         extra_fields={"description": "Also includes data usage information"}),
+    Link(
+        "product handbook",
+        "https://spacedata.copernicus.eu/documents/20123/122407/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I5.0+%281%29.pdf/706ee17d-2cce-f1fa-a73e-1686d28f09dd?t=1679657087883"  # noqa: E501
+        "application/pdf",
+        "Compernicus DEM Product Handbook (Nov 2022)",
+        extra_fields={"description": "Also includes data usage information"}),
     Link(
         "license",
         "https://spacedata.copernicus.eu/documents/20123/121286/CSCDA_ESA_Mission-specific+Annex_31_Oct_22.pdf",  # noqa: E501
         "Copernicus Data Access")
 ]
 
 COP_DEM_ASSETS = {
-    "data": AssetDefinition({
+    "data":
+    AssetDefinition({
         "type": MediaType.COG,
         "roles": ["data"],
-    })
+    }),
+    "edm":
+    AssetDefinition({
+        "title": "Editing mask",
+        "type": MediaType.COG,
+        "description": "Editing Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "flm":
+    AssetDefinition({
+        "title": "Filling mask",
+        "type": MediaType.COG,
+        "description": "Filling Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "wbm":
+    AssetDefinition({
+        "title": "Water Body mask",
+        "type": MediaType.COG,
+        "description": "Water Body Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "hem":
+    AssetDefinition({
+        "title": "Height Error mask",
+        "type": MediaType.COG,
+        "description": "Height Error Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "acm":
+    AssetDefinition({
+        "title": "Height Error mask",
+        "type": "application/vnd.google-earth.kml+xml",
+        "description": "Height Error Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "src":
+    AssetDefinition({
+        "title": "Source mask",
+        "type": "application/vnd.google-earth.kml+xml",
+        "description": "Source Mask",
+        "role": ["data", "data-mask"],
+    }),
+    "dem_ql":
+    AssetDefinition({
+        "title": "DEM QuickLook Relative",
+        "type": MediaType.TIFF,
+        "description":
+        "Quicklook of Digital Elevation Model wiuth shaded relief representation (relative)",
```

Comment:
```suggestion
        "QuickLook of Digital Elevation Model with shaded relief representation (relative)",
```
Based off Chuck's comments - should make the change here and rerun examples